validator/schema/upsun.json

```diff
@@ -23,20 +25,32 @@
             },
             "additionalProperties": false,
             "nullable": true,
-            "title": "Resources",
+            "title": "Use flexible resources.",
+            "description": "Adjust the amount of memory using the resources key in your Platform.sh app configuration. All memory must stay within the limits set by your plan.",
             "default": null
           },
           "container_profile": {
-            "type": "string",
+            "type": ["string", "null"],
             "title": "The container profile for this application in production. Leave blank to allow it to be set dynamically.",
-            "default": "AUTO"
+            "description": "More information:\\nhttps://docs.upsun.com/manage-resources.html",
+            "default": null,
+            "anyOf": [
+              {
+                "type": "string",
+                "enum": ["HIGH_CPU", "BALANCED", "HIGH_MEMORY", "HIGHER_MEMORY"]
```

Comment:
Nice addition, lets add the same `container_profile` to services too its supported on upsun cf https://docs.upsun.com/manage-resources/adjust-resources.html#adjust-a-container-profile

Reply:
better than that, i did some refactoring and introduce some `$ref` for the `container_profile` (defined at the end of the file)